GVRf/Sample/gvrjassimpmodelloader/src/org/gearvrf/jassimpmodelloader/JassimpModelLoaderViewManager.java

```diff
@@ -95,6 +96,14 @@ public void run() {
         mMainScene.addSceneObject(benchModel);
 
         rotateModel(astroBoyModel, 10f, astroBoyModelPosition);
+
+        // Model over network
+        String urlBase = "https://raw.githubusercontent.com/danke-sra/GearVRf/jassimp_network/GVRf/Sample/gvrjassimpmodelloader/assets/";
```

Comment:
I believe this needs to be updated to point to the (expected) corresponding location in the Demos repository.